src/types.h

```diff
@@ -65,11 +65,11 @@ template<typename T> T rad2deg(T val) { return T(val * r2d); }
 #include "../common/android_helpers.h"
 #endif
 
+#define UNKNOWN_VALUE "UNKNOWN"
+
 namespace librealsense
 {
-    #define UNKNOWN_VALUE "UNKNOWN"
-
-    static const float TIMESTAMP_USEC_TO_MSEC = 0.001;
+    static const float TIMESTAMP_USEC_TO_MSEC = 0.001f;
```

Comment:
I've modified this one in https://github.com/IntelRealSense/librealsense/pull/3773
Change it manually from float to double for consistency